builtin/logical/consul/secret_token.go

```diff
@@ -67,6 +67,7 @@ func secretTokenRevoke(ctx context.Context, req *logical.Request, d *framework.F
 	}
 
 	tokenRaw, ok := req.Secret.InternalData["token"]
+	version, ok := req.Secret.InternalData["version"]
```

Comment:
This previously checked the tokenRaw but now checks the version.  I think if version is not populated, you should just use the ACL destroy method.